client/public/index.html

```diff
@@ -8,8 +8,8 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="icon" href="2011_Logo_white.png" />
-    <link rel="apple-touch-icon" href="2011_Logo_white.png" />
+    <link rel="icon" href="WTix logo light.png" />
+    <link rel="apple-touch-icon" href="WTix logo light.png" />
```

Comment:
Probably a non-issue but not having any spaces in the image file name would be on the safer side